content/docs/install/index.md

```diff
@@ -16,12 +16,12 @@ if needed. The [Python API](/doc/api-reference) module is `dvc.api`.
 
 ## Advanced options
 
+- You can [install a stable pre-release](/doc/install/pre-release) of DVC to
```

Comment:
we should probably update that doc to include an instruction to install pip pre-release (command is in the last blog post)

Reply:
Done in a3a70895 I think